pkg/feature/feature.go

```diff
@@ -0,0 +1,40 @@
+package feature
+
+import (
+	"github.com/spf13/pflag"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/component-base/featuregate"
+)
+
+const (
+	// MyFeature featuregate.Feature = "MyFeature"
+	// owner: @username
+	// alpha: v1.X
+	// (see https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go)
+
+	// OperatorLifecycleManagerV2 enables OLM's v2 APIs.
+	// owner: @njhale
+	// alpha: v0.15.0
+	OperatorLifecycleManagerV2 featuregate.Feature = "OperatorLifecycleManagerV2"
```

Comment:
We probably want to be more granular than this. i.e. `OperatorV1`

Reply:
My thinking here is that we're probably going to enable all of the v2 APIs at once when we jump to v2. Do you see a situation where we want to enable APIs independently? i.e. `InstallV2` w/o `OperatorV2`?